oat-sa/tao-extension-release
# Breaking/remove manifest support

- Refactoring: remove version writing in extension's manifest to support new extension versioning model based on packagist (git tags) 
 - this lead to removing all code linked to php parsing and extension release management based on manifest version
 - remove some unused code previously used in 2 steps releases only 
 - remove the pull request checklist since there should be no need for manual verifications any more

 How to test: 
 - try to release extension-tao-foobar (you'll see the manifest version is now disconnected from the tags and shouldn't be updated)
 - run the unit tests
